fe1-web/src/features/home/interface/Feature.ts

```diff
@@ -0,0 +1,13 @@
+export namespace HomeFeature {
+  export interface LaoState {}
+
+  export interface Lao {
+    toState: () => LaoState;
+  }
+
+  export interface MainNavigationScreen {
+    name: string;
+    Component: React.ComponentType<unknown>;
+    order: number;
```

Comment:
I would be wary of using `order` as a number without further recommendation.
Developers' natural tendency will be to number the screens 0, 1, 2, 3, ...
This makes it very hard to "insert" something in the middle, and leads to subtle, coupled changes (i.e. having to change some code in all the features to insert one in the middle).
I would suggest thinking of it as one normally does with z-index , where any number, positive or negative, is allowed.
As such, items might be ordered (why not!) with numbers `-9999, 0, 10, 20, 9999` 

Furthermore, I don't see logic that validates that no two screens have the same order, which can lead to subtle bugs.
I would a few comments to document the use of this value, and I'd try to add runtime checks to make sure it actually gets used correctly: fail fast, fail hard 😉

Reply:
That is true!! I will add that, thank you for the feedback :)

(Although given it is javascript you could write `0.5`.. 😜😂)